### PR TITLE
Stats persistence

### PR DESCRIPTION
**Pull request by Bagaric**
screen shot 2018-08-22 at 00 24 11

DEPENDS ON https://github.com/ethereum-alarm-clock/timenode-core/pull/230 BEING RELEASED.

- Adjust the CLI to the new StatsDB refactors
- Makes the stats persistent
- Add `resetStats` command
- Add a dev command `cleanup` for cleaning files created by running the CLI TimeNode

### CODE REVIEW

File: src/TimeNode/index.js [hunk 1]
```diff
@@ -44,6 +45,23 @@ const timenode = async (options, program) => {
...
   });
 
+  const statsDb = new loki('stats.json', {
+    adapter: new lfsa(),
+    autoload: true,
+    autoloadCallback: () => {
+      // LokiJS stores BN objects as a string.
+      // This causes problems when loading from persistent storage.
+      // Convert any stored non-BN into BN.
+      const stats = statsDb.getCollection('timenode-stats').data;
```
Comment (kosecki123) [inline src/TimeNode/index.js]: Where exactly the timenode-stats files is stored? Maybe it should be in ~ directory next to othet .files

Comment (kosecki123) [reply src/TimeNode/index.js]: That's a good question, especially when you consider linux based systems and windows. I have not good answer now :|

**Comment by kosecki123** [comment]
`timenode-core` 5.0.1 has been released.

File: src/TimeNode/index.js (hunk 1 above)
Comment (Bagaric) [reply src/TimeNode/index.js]: @kosecki123 as @lsaether said, the file is stored in whichever directory the TimeNode process is started from. I agree we should change it to an absolute path.

**Pull request merged**